backend/src/main/java/gov/cdc/usds/simplereport/api/model/AoEQuestions.java

```diff
@@ -0,0 +1,78 @@
+package gov.cdc.usds.simplereport.api.model;
+
+import java.time.LocalDate;
+
+public class AoEQuestions {
```

Comment:
We have found that adding API models creates a lot of boiler plate and have found the `GraphQLResolver` to be a better pattern. See https://github.com/CDCgov/prime-simplereport/blob/main/backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationDataResolver.java

the `GraphQLResolver` will automagically pull the getters from the model.

Reply:
This API model is used for the REST endpoint – how would I leverage the GraphQL Resolver in that context to deserialize this nested JSON data bag?